### PR TITLE
Erssup 65487

### PR DESCRIPTION
**Pull request by KevinMayfield**
Applied changes as agreed and listed on https://nhsd-confluence.digital.nhs.uk/display/ES/2023-05-04

### CODE REVIEW

File: CodeSystem/CodeSystem-eRS-RequestFlowType.json
Comment (adamoldfield-nhs) [inline CodeSystem/CodeSystem-eRS-RequestFlowType.json:1]: As per https://nhsd-confluence.digital.nhs.uk/display/ES/2023-05-04:

> New Code System: https://fhir.nhs.uk/CodeSystem/eRS-BookableType
> * Same values as https://fhir.nhs.uk/STU3/CodeSystem/eRS-BookableType-1 but no UNSPECIFIED
> Update [eRS-HealthcareService](https://simplifier.net/packages/uk.nhsdigital.r4/2.8.0/files/1953527) profile to include example binding to new BookableType
> * 0..1 binding

So I don't there should be any changes to the RequestFlowType.

File: HealthcareService/ers-HealthcareService-searchResults.json
Comment (adamoldfield-nhs) [inline HealthcareService/ers-HealthcareService-searchResults.json:1]: As per comment on RequestFlowType I don't think there should have been any changes here.

File: StructureDefinition/Extension-eRS-HealthcareService-Contact.json [hunk 1]
```diff
@@ -101,8 +101,7 @@
         "sliceName": "telecom",
         "short": "The contact point details",
         "definition": "The contact point details.",
-        "min": 1,
-        "max": "1"
```
Comment (adamoldfield-nhs) [inline StructureDefinition/Extension-eRS-HealthcareService-Contact.json:105]: Assuming removing the setting of `max` defaults to * then this looks right.

**Review by DaveBarnet — APPROVED** [review]
CodeSystem update looks OK as agreed with eRS, like Adam, not sure what the change to NHSDigital-HealthcareService structureDefinition does?

**Pull request merged**